src/renderer/constants/icons.js

```diff
@@ -1,5 +1,5 @@
-export const FEATURED = 'rocket';
-export const LOCAL = 'folder';
+export const FEATURED = 'Award';
```

Comment:
Going to add every icon to this file and only change every Icon use to constants

Reply:
* at a later time

Reply:
is there a reason why the ones you just added are upper case but the others are lower?